### PR TITLE
Update Treasure Trail tool

### LINKED ISSUE #3: Update treasure trails tool
A few months ago, there was treasure trail expansion. This treasure trails tool need to be updated for this game update and the new items added.

### PR DESCRIPTION
**Pull request by AntiTcb**
Resolves #3. 

Also added in Clueless Scroll Medium reward, a shared reward space for god book pages, master scroll book, and Ring of 3rd age. Potentially may want to move most 3rd age pieces here since they can be rewarded from hard, elite, and masters now.

### CODE REVIEW

File: tools/treasure-trails/index.md [hunk 1]
```diff
@@ -206,111 +243,121 @@ include_custom_script:
...
+                <div class="col py-1"><span class="osrstooltip" data-type="short" id="23392" title="Please wait ..."><img id="23392-img" class="tt-notselected" src="https://www.osrsbox.com/osrsbox-db/items-icons/23392.png" alt="alt text" onclick="javascript:update_rewards('23392','medium')"></span></div>
+                <div class="col py-1"><span class="osrstooltip" data-type="short" id="23395" title="Please wait ..."><img id="23395-img" class="tt-notselected" src="https://www.osrsbox.com/osrsbox-db/items-icons/23395.png" alt="alt text" onclick="javascript:update_rewards('23395','medium')"></span></div>
+                <div class="col py-1"><span class="osrstooltip" data-type="short" id="23398" title="Please wait ..."><img id="23398-img" class="tt-notselected" src="https://www.osrsbox.com/osrsbox-db/items-icons/23398.png" alt="alt text" onclick="javascript:update_rewards('23398','medium')"></span></div>
+                <div class="col py-1"><span class="osrstooltip" data-type="short" id="23401" title="Please wait ..."><img id="23401-img" class="tt-notselected" src="https://www.o
```
Comment (osrsbox) [inline tools/treasure-trails/index.md]: Just one small note: the `Clueless scroll` was already in the Medium clue scroll list - so adding it here has made a duplicate in the list. The original one is between the `Gnomish firelighter` and the `Crier hat`. Happy to accept the PR as-is and remove the line myself, or if you want to push the changed to this PR? Just let me know.

File: tools/treasure-trails/rewards.js [hunk 2]
```diff
@@ -225,6 +253,16 @@ medium.set(20254, false); // Hosidius banner
...
+medium.set(23389, false); // Spiked manacles
+medium.set(23410, false); // Wolf cloak
+medium.set(23407, false); // Wolf mask
+medium.set(23413, false); // Climbing boots (g)
+medium.set(23392, false); // Adamant platebody (h1)
+medium.set(23395, false); // Adamant platebody (h2)
+medium.set(23398, false); // Adamant platebody (h3)
+medium.set(23401, false); // Adamant platebody (h4)
+medium.set(23404, false); // Adamant platebody (h5)
+medium.set(20249, false); // Clueless scroll
```
Comment (osrsbox) [inline tools/treasure-trails/rewards.js]: Same note as above, the `Clueless scroll` was already in the list.

File: tools/treasure-trails/index.md (hunk 1 above)
Comment (AntiTcb) [reply tools/treasure-trails/index.md]: Interesting, I even Ctrl + F'd for it to see if it was there and I was just missing it. I'll go ahead and remove the duplicate and move the other 3rd age items to the shared section later tonight.

Comment (osrsbox) [reply tools/treasure-trails/index.md]: Awesome - many thanks! Just did a double check and it is in the `index.html` page twice (found using `20249` item ID).... but it is not in the `rewards.js` file twice. Thanks very much for this PR - it is very much appreciated :)

**Comment by AntiTcb** [comment]
Decided against moving 3rd age (and gilded) to shared for now, felt like too obtrusive of a change, alongside that changing the corresponding js object from their original tier to `shared` would be a breaking change for loading in already saved data. Also couldn't decide (or style) if there should be sub-groups or help text explaining exactly which tiers the items are shared with, so I'll leave it currently as the ones that any tier (except beginner) can access, along with the Ring of 3rd age, since those just weren't previously on the page at all.

**Comment by osrsbox** [comment]
Thanks again for this PR @AntiTcb and the work you put in. It is much appreciated :1st_place_medal:

**Pull request merged**